scripts/backend_test_shards.json

```diff
@@ -236,6 +236,7 @@
         "core.platform.taskqueue.cloud_taskqueue_services_test",
         "core.platform.cloud_translate.cloud_translate_services_test",
         "core.platform.cloud_translate.dev_mode_cloud_translate_services_test",
+        "core.platform.logging.logging_service_test",
```

Comment:
The backend test shard 11 is taking more than 30 minutes to run on this PR:

![Screen Shot 2021-05-31 at 23 39 03](https://user-images.githubusercontent.com/19878639/120263327-7c6ffa00-c269-11eb-8338-5ce11598139f.png)

We want each shard to run in <30 mins to keep the tests fast on contributors' PRs, so could you add this test to a different shard please? You can just pick the shard with the shortest runtime